service/docs/source/geopm_pio_nvml.7.rst

```diff
@@ -0,0 +1,205 @@
+
+
+geopm::NVMLIOGroup(7) -- IOGroup providing signals and controls for NVIDIA GPUs
+=================================================================================================
+
+Description
+-----------
+
+The NVMLIOGroup implements the `geopm::IOGroup(3) <GEOPM_CXX_MAN_IOGroup.3.html>`_
+interface to provide hardware signals and controls for NVIDIA GPUs.
+
+Requirements
+~~~~~~~~~~~~
+To use the GEOPM NVML signals and controls GEOPM must be compiled against the NVML libraries and must be run on a system with hardware supported by NVML.
```

Comment:
Same comment here as with the L0 docu: we should spell out what it means to "compile GEOPM against NVML" for the configure time options and pointers to other relevant build time docu.